README.md

```diff
@@ -122,6 +122,13 @@ Please refer to the demos of the MQTT client library in the following locations
 
 ## Generating documentation
 
+Note: For pre-generated documentation, please see the documentation linked in the above repositories:
+
+| Location |
+| :-: |
+| [AWS IoT Device SDK for Embedded C](https://github.com/aws/aws-iot-device-sdk-embedded-C#releases-and-documentation) |
+| [FreeRTOS.org](https://freertos.org/Documentation/api-ref/coreMQTT/docs/doxygen/output/html/index.html) |
```

Comment:
If the released versions of C-SDK and FreeRTOS.org don't match, it would be confusing to customers about the reason of their disparity. It would be important to add a note saying that it is possible for the C-SDK and FreeRTOS releases to contain different release versions of coreMQTT